VWOSdk/Internal/Core/SegmentEvaluator.cs

```diff
@@ -75,6 +75,21 @@ public dynamic getTypeCastedFeatureValue(dynamic value, string variableType)
                 {
                     return Convert.ToBoolean(value);
                 }
+                if (variableType == Constants.VariableTypes.JSON)
```

Comment:
why is it required in SegmentEvaluator file?

Reply:
This function is called by GetFeatureVariableValue for feature testing . it validate the json variable data.